### DISCUSSION TITLE
Cloud Carbon Footprint has enabled an optional usage for Electricity Maps

### DISCUSSION THREAD

**Discussion by camcash17** [electricitymaps/electricitymaps-contrib#5895] [General]
Hello!

As part of the CCF community, we are excited to announce in our [latest release](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/releases/tag/latest), we have implemented the option for users to consume Electricity Maps carbon intensity data to make the CCF cloud emissions estimations!

We have provided a bit of documentation [here](https://www.cloudcarbonfootprint.org/docs/emissions-factors#electricity-maps-api-configuration), and we would welcome any feedback or suggestions on our approach.

Thanks for the great work!

**Comment by VIKTORVAV99** [comment]
cc @madsnedergaard @pierresegonne
